### ISSUE TITLE
Dirt Rally 1 + DSX v3.1 (v1.38 beta) + RacingDSX v0.64 - No effect on triggers

### ISSUE THREAD

**Issue by anijatsu** [cosmii02/RacingDSX#55]
Hi, 
I've got an odd issue where RacingDSX seems to pick up the telemetry data from DiRT Rally 1 just fine, but then DSX either ignores it, or cannot receive it properly.
I'm attaching a few screenshots that might narrow it down:
DSX v3.1 does have a panel called Live Incoming Data Feed, and an error pops up there:
```
GetDSXStatus (Made for getting Server Response)
Raw Json Data - {}
Failed to display Mod Data. Potential format error. Please contact mod author or ask for help in the DSX Discord server.
```
![Screenshot 2024-12-05 203910](https://github.com/user-attachments/assets/a3a39fb2-0a84-442f-8439-5df61b04b6d8)

For testing purposes I set the minimum throttle and brake resistance to 6, and the output panel running on full verbose mode confirms that it should be setting it as such:
![Screenshot 2024-12-05 204046](https://github.com/user-attachments/assets/5f59d769-0380-46ed-a12e-37b6a91217ac)
![Screenshot 2024-12-05 204419](https://github.com/user-attachments/assets/61ae1576-8f0a-420b-8826-54f4b46d1b07)

What's potentially suspicious is that the top pane of the window is almost always empty. Any clue how to debug this further?
I have following .NET runtimes installed:
![image](https://github.com/user-attachments/assets/04f43177-c7b8-4b2f-82e2-e9fe7cdf9a9e)

**Comment by cosmii02** [comment]
After testing, there's something wrong with how json data is handled, for now you should use 0.6.2 until the issue gets fixed

**Comment by cosmii02** [comment]
@anijatsu The issue should be fixed with 0.6.5
